week 5/README.md

```diff
@@ -190,3 +190,25 @@ Output
 1 2 3
 ```
 
+
+### Task 3 - WTF Cipher
```

Comment:
1. These tasks are meant to be exercised without grading. It doesn't matter if they find the solution. It's even better if they find the solution and at least try to understand it instead of throwing away the problem should they find it too difficult.
2. How about?

> The best way to test whether your encoding algorithm is working is to add a decoding algorithm that reverses the operation. Your program should first ask whether to encode or decode the given strings